Sources/TuistGraph/Models/CoreDataModel.swift

```diff
@@ -13,6 +13,8 @@ public struct CoreDataModel: Equatable, Codable {
 
     /// Current version without the extension.
     public let currentVersion: String
+    
+    public let platformFilters: [PlatformFilter]?
```

Comment:
Can you please also add swift doc as for the other fields?

```suggestion
    public let platformFilters: [PlatformFilter]
```